vllm/model_executor/models/phi3v.py

```diff
@@ -672,3 +683,11 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                     weight_loader = getattr(param, "weight_loader",
                                             default_weight_loader)
                     weight_loader(param, loaded_weight)
+
+        # We use regex to extract the sub-module name
+        # from "model.vision_embed_tokens.img_processor.*"
+        vision_weights = [
+            (re.search(r"vision_embed_tokens\.img_processor\.(.*)",
+                       n).group(1), w) for n, w in vision_weights
+        ]
+        self.vision_embed_tokens.img_processor.load_weights(vision_weights)
```

Comment:
As a TODO, we should probably add an `prefix` argument for `CLIPVisionModel.load_weights` so that we don't need to hack it this way. What do you think?

Reply:
That would be a great fix. For some models like Phi3V, it was hard to use the `load_weights` function of the vision model just like `LlavaForConditionalGeneration`.

Reply:
Feel free to add this as a TODO on top of `load_weights`, we don't have to do it in this PR!